rollupsync/batch_data.go

```
@@ -0,0 +1,228 @@
+package rollupsync
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"time"
+
+	authzv1beta1 "cosmossdk.io/api/cosmos/authz/v1beta1"
```

golangci-lint (1.62.2)

import 'cosmossdk.io/api/cosmos/authz/v1beta1' is not allowed from list 'main'

(depguard)